src/web/layouts/StandardLayout.tsx

```diff
@@ -559,24 +566,25 @@ export const StandardLayout = ({
             </Section>
 
             <Section
+                data-print-layout='hide'
```

Comment:
if the parent is hidden, wouldnt you not need to have `data-print-layout='hide'` in every child?